service/src/main/java/org/cbioportal/genome_nexus/service/internal/CachedVariantAnnotationFetcher.java

```diff
@@ -0,0 +1,24 @@
+package org.cbioportal.genome_nexus.service.internal;
+
+import org.cbioportal.genome_nexus.model.VariantAnnotation;
+import org.cbioportal.genome_nexus.persistence.VariantAnnotationRepository;
+import org.cbioportal.genome_nexus.persistence.internal.VariantAnnotationRepositoryImpl;
+import org.cbioportal.genome_nexus.service.remote.VEPDataFetcher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CachedVariantAnnotationFetcher extends BaseCachedExternalResourceFetcher<VariantAnnotation, VariantAnnotationRepository>
+{
+    @Autowired
+    public CachedVariantAnnotationFetcher(ExternalResourceTransformer transformer,
```

Comment:
We will need to add a class for each resource we want to cache. Implementation is pretty straightforward, just autowiring and setting the concrete values.